src/test/ui/suggestions/try-operator-on-main.stderr

```diff
@@ -1,14 +1,37 @@
-error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
-  --> $DIR/try-operator-on-main.rs:12:5
+error: use of unstable library feature 'try_trait' (see issue #42327)
+  --> $DIR/try-operator-on-main.rs:11:5
+   |
+11 | use std::ops::Try;
+   |     ^^^^^^^^^^^^^
+   |
+   = help: add #![feature(try_trait)] to the crate attributes to enable
+
+error: use of unstable library feature 'try_trait' (see issue #42327)
+  --> $DIR/try-operator-on-main.rs:19:25
+   |
+19 | fn try_trait_generic<T: Try>() {}
+   |                         ^^^
    |
-12 |     std::fs::File::open("foo")?;
+   = help: add #![feature(try_trait)] to the crate attributes to enable
+
+error[E0277]: the `?` operator can only be used in a function that returns `Result` (or another type that implements `std::ops::Try`)
+  --> $DIR/try-operator-on-main.rs:14:5
+   |
+14 |     std::fs::File::open("foo")?;
    |     ---------------------------
    |     |
-   |     the `?` operator can only be used in a function that returns `Result` (or another type that implements `std::ops::Try`)
+   |     the trait `std::ops::Try` is not implemented for `()`
```

Comment:
I am worried that the error text is less prominent than the label, so moving the more relevant text to the error message makes it less useful for newbies. But this might just be an unfounded personal reaction.

Maybe adding an optional `label` attribute to `rustc_on_unimplemented` for text to be displayed in the label would be reasonable? (If I understood the code above correctly, the feature is already there, it just needs to be changed in the trait, right?)